dev-libs/spdlog/spdlog-1.5.0.ebuild

```diff
@@ -21,19 +21,29 @@ SLOT="0/1"
 IUSE="test"
 RESTRICT="!test? ( test )"
 
+BDEPEND="
+	virtual/pkgconfig
+"
 DEPEND="
-	>=dev-libs/libfmt-5.0.0
+	>=dev-libs/libfmt-5.3.0:=
 "
 RDEPEND="${DEPEND}"
 
-PATCHES=( "${FILESDIR}/${PN}-1.0.0-unbundle-fmt.patch" )
+#PATCHES=(
+#	"${FILESDIR}/${PN}-1.4.2-fix-compilation.patch"
+#)
```

Comment:
Why is this commented out?

If it's not applicable, I should be removed (and not left as a comment)

Reply:
Sorry, I was testing if that patch was still needed and forgot to remove the comment.